frontend/packages/dev-console/integration-tests/features/Knative/filters-Serving-Eventing-Admin.feature

```diff
@@ -0,0 +1,83 @@
+        When user clicks on dropdown button
+        And user selects Name
+        And user enters "hello-openshift"
+        Then user will see KSVC by name "hello-openshift"
+        And user will see Clear all filters
+
+
+    @regression
+    Scenario: Search by label correctly
+        Given user is at Serving page
+        And user has created knative service "hello-openshift"
+        And user has selected Routes tab
+        When user clicks on dropdown button
+        And user selects Label
+        And user enters "hello-openshift"
+        Then user will see routes for KSVC by name "hello-openshift"
+        And user will see Clear all filters
+        
+        
+    @regression
+    Scenario: Search by name incorrectly
+        Given user is at Serving page
+        And user has created knative service "hello-openshift"
+        And user has selected Revisions tab
+        When user clicks on dropdown button
+        And user selects Name
+        And user enters "xyz"
+        Then user will see message "No Revisions found"
+        And user will see Clear all filters
+
+
+    @regression
+    Scenario: Event Sources Filter
+        Given user has created knative service "hello-openshift"
+        And user has created ApiServer Source
+        And user has created Ping Source
+        And user has created Sink Binding
+        And user has created Container Source
+        And user is at Event Sources tab
+        When user clicks on Filter dropdown
+        Then user will see digit "1" in front of ApiServer, Ping, Sink, Container Sources checkboxes
+        And user will see only ApiServer source when checked checkbox in front of ApiServer Source
+        And user will see only Container source when checked checkbox in front of Container Source
+        And user will see only Ping source when checked checkbox in front of Ping Source
+        And user will see only Sink source when checked checkbox in front of Sink Source
+        
+
+    @regression
+    Scenario: Event Sources Filter
+        Given user has created knative service "hello-openshift"
+        And user has created Default Memory Channel
+        And user has created In Memory Channel
+        And user is at Channels tab
+        When user clicks on Filter dropdown
+        Then user will see digit "1" in front of Channel and In Memory Channel checkboxes
+        And user will see only Channel when checked checkbox in front of Channel
+        And user will see only In Memory Channel when checked checkbox in front of In Memory Channel
+    
+
+    @regression, @manual
+    Scenario: Event Sources Filter
+        Given user has created knative service "hello-openshift"
+        And user has created Github source
+        And user has created Ping Binding
+        And user is at Event Sources tab
+        When user clicks on Filter dropdown
+        Then user will see digit "1" in front of Another Type, Ping Sources checkboxes
```

Comment:
Verify this behaviour once , as long as we are able to identify the source we can show that in type. Other type was mentioned as they can't show all types in design cc @sahil143

Reply:
Updated